PurchasesCoreSwift/Attribution/TrackingManagerProxy.swift

```diff
@@ -56,11 +53,11 @@ public class TrackingManagerProxy: NSObject {
         NSClassFromString(mangledTrackingClassName.rot13())
     }
 
-    @objc public var authorizationStatusPropertyName: String {
+    var authorizationStatusPropertyName: String {
         Self.mangledAuthStatusPropertyName.rot13()
     }
 
-    @objc func trackingAuthorizationStatus() -> Int {
+    func trackingAuthorizationStatus() -> Int {
```

Comment:
That was hard to figure out. Not sure why, but this change was breaking some tests.